spec/javascripts/ModalEditorSpec.js

```diff
@@ -0,0 +1,25 @@
+describe("ModalEditor", function() {
```

Comment:
Regarding the issue you encountered with the tests failing on jasmine:ci, maybe resetting the modal after each it block is run will fix it? But you do have a beforeEach already 🤔